Projects/hCoreUI/Sources/hForm/hRow.swift

```diff
@@ -85,7 +85,7 @@ public struct hRow<Content: View, Accessory: View>: View {
     public var body: some View {
         VStack(alignment: .leading, spacing: 0) {
             VStack(alignment: .leading) {
-                HStack(alignment: .top) {
+                HStack(alignment: .center) {
```

Comment:
Nice thought! I don't think we should not change hRow though since the changes will be reflected everywhere where hRow is used. Instead, we should edit in that affected view (AppInfoView in this case)

Reply:
you could for example add `.frame(maxHeight: .infinity, alignment: .center)` to the hText element there 😄

Reply:
`                hText(deviceId, style: .standardSmall)
                    .minimumScaleFactor(0.2)
                    .lineLimit(1)
                    .foregroundColor(hTextColor.Opaque.secondary)
                    .frame(maxHeight: .infinity, alignment: .center)`